packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -164,9 +166,16 @@ export type AliasResult = "Success" | "Conflict" | "AlreadyAliased";
 
 /**
  * Exposes some functionality/features of a data store:
+ *
  * - Handle to the data store's entryPoint
+ *
  * - Fluid router for the data store
+ *
  * - Can be assigned an alias
+ *
+ * @privateRemarks
+ * These docs should define what a datastore is, and not do so by just referencing "data store".
```

Comment:
```suggestion
 * TODO: These docs should define what a datastore is, and not do so by just referencing "data store".
```

Reply:
@ChumpChief can you offer additional context here? Do you know who can?

Reply:
(I'm perfectly happy to just have the TODO checked in for now, but if we _can_ just improve the docs now...)

Reply:
I'd probably leave it as todo for now and ask around.  When I say data store, I personally mean "a DataObject or something kinda like it".  In this usage it seems to be trying to be more particular though, and I don't get the nuance.

Also "data store" and "data object" are garbage terms that mean nothing - like most of the stuff in memory could justifiably be called "data" or an "object".  Would be great if in the course of the interface cleanup if we could find better names that actually mean something.